src/NpmUtilities.js

```diff
@@ -145,11 +146,17 @@ export default class NpmUtilities {
     );
   }
 
-  static publishTaggedInDir(tag, directory, registry, callback) {
+  static publishTaggedInDir(tag, directory, registry, config, callback) {
     log.silly("publishTaggedInDir", tag, path.basename(directory));
-
     const opts = NpmUtilities.getExecOpts(directory, registry);
-    ChildProcessUtilities.exec("npm", ["publish", "--tag", tag.trim()], opts, callback);
+    let cmd = 'npm';
+    if (typeof config === 'function') {
```

Comment:
All these conditionals are too complex. We literally control all callers of this method, if the signature changes then we can assume we can change it everywhere it is called.